docs/Software Architecture Document/Components View/index.md

```diff
@@ -148,6 +148,21 @@ The view is represented by module and subsystem diagrams that show the system's
     - Not work for Android apps.
     - Not work for apps connected via SDL protocol version lower than 4.
 
+#### Plugin Manager
```

Comment:
@pestOO code design is already done by RevSDL team. We will change it's implementation in future, I suppose that during changing architecture we will give more appropriate names.
Also, See constrains e1b6dea0bbf96d91a52457dab8a8bcba367db144